.travis.yml

```diff
@@ -101,8 +101,7 @@ jobs:
         - yarn autosign:drive:android
       after_success:
         - *downcloud-cert
-        - APK_URL=$(node scripts/downcloud.js src/drive/targets/mobile/build/android/cozy-drive.apk)
-        - scripts/github.sh "🎁 [Click here]($APK_URL) to download the latest Android APK"
+        - if [[ "$(git log -2)" =~ [\[APK\]] ]]; then APK_URL=$(node scripts/downcloud.js src/drive/targets/mobile/build/android/cozy-drive.apk) && scripts/github.sh "🎁 [Click here]($APK_URL) to download the latest Android APK"; fi
```

Comment:
Can't we use `TRAVIS_COMMIT_MESSAGE`?
Also, `git log -2` means that we look at the last 2 commits, right? (in the description of the PR, you say : "present in the last commit message").
nit: It would be nice to have the 2 commands (upload and commit) in distinct lines like before but I'm not sure how we can do this

Reply:
Should have clarified this in the PR description, but no, we can't use `TRAVIS_COMMIT_MESSAGE`. Travis runs with a simulated merge of the PR branch into master — for example https://github.com/cozy/cozy-drive/commit/665c4c83f8fbad475a1157ab8f2bb5bae044ff23. So instead we want the commit message *just before* that one, ie.  `git log -2`.

Reply:
ok ! It's not cool that travis does not provide the real commit message somewhere!

Reply:
Yeah, it depends on when and how the tests are run, but this isn't ideal. Luckily it's still a git repo :)